public/stylesheet.css

```diff
@@ -748,6 +748,7 @@ body.order-by-relevance .grid-item {
   .footer .footer-share {
     flex-grow: 1;
     justify-content: center;
+    margin-bottom: 0.3rem;
```

Comment:
I feel like these are still a bit close together on smaller screens: 

![image](https://user-images.githubusercontent.com/3742559/132980155-1830991e-7df6-4978-a10b-a4c4eab9e4dc.png)

Maybe we can increase the value a bit here, e.g.

```suggestion
    margin-bottom: 0.8rem;
```

Reply:
Hmm :thinking: seeing a similar problem at a slightly larger size (this is exactly 682px wide):

![image](https://user-images.githubusercontent.com/3742559/133157874-dac845b6-2b9c-44bd-921d-8bcb245846f2.png)

Reply:
@jorgeamadosoria, did you see this comment?

Reply:
no. I just got notified now.

Reply:
fixed